sources/assets/Stride.Core.Assets.Quantum/AssetCompositeHierarchyPropertyGraph.cs

```diff
@@ -624,6 +624,12 @@ private void UpdateAssetPartBases()
             if (part.Base is null)
                 continue;
 
+            if (part.Base.BasePartAsset is null)
+                continue;
+
+            if ((Guid)part.Base.BasePartAsset.Id == Guid.Empty)
```

Comment:
I'm unsure about that one. Needs more testing so that it doesn't break anything else.

Reply:
I wrote this because I wasn't sure how Stride handles empty references. I expected that this could eventually happen, if somehow a invalid object gets read again.

Removing this had no negative effect when testing the same steps as initially described.

Reply:
See the latest commit.